scripts/check_compliance.py

```diff
@@ -578,33 +578,51 @@ def run(self):
 
         report = ""
 
-        whitelist_extensions =  ['.yaml', '.html']
-        whitelist_languages = ['CMake', 'HTML']
+        whitelist_extensions =  ['.yaml', '.html', '.rst', '.conf', '.cfg']
+        whitelist_languages = ['HTML']
+        blacklist_languages = ['CMake']
```

Comment:
Bit tricky (for me at least) to understand the intent here.

What does it mean if a language is in neither the whitelist or the blacklist? What if it's in both? Maybe a comment could be added to explain it.

Is there a more explicit name than "whitelist"/"blacklist" that could be used (like `always_checked_langs` and `never_checked_langs` or the like)? I keep getting confused about whether "whitelist" means to "whitelist it for the test" or "whitelist it so that the test doesn't check it". :)